maven-plugin/src/test/java/io/confluent/kafka/schemaregistry/maven/DownloadSchemaRegistryMojoTest.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafka.schemaregistry.maven;
+
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
+import org.apache.avro.Schema;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DownloadSchemaRegistryMojoTest {
+  File tempDirectory;
+  DownloadSchemaRegistryMojo mojo;
+
+  @Before
+  public void before() throws IOException {
+    this.tempDirectory = File.createTempFile(this.getClass().getSimpleName(), "tmp");
+    this.tempDirectory.delete();
+    this.tempDirectory.mkdirs();
+    this.mojo = new DownloadSchemaRegistryMojo();
+    this.mojo.client(new MockSchemaRegistryClient());
+  }
+
+  @Test
+  public void download() throws IOException, RestClientException {
```

Comment:
Not sure I understand what this is supposed to be testing? I thought the `DownloadSchemaRegistryMojoTest` would test functions of the `DownloadSchemaRegistryMojo`, but this just seems to invoke the client to registry schemas (which also doesn't seem to match the name of the method)?

Same holds for the rest of the tests in this class -- none ever seem to execute the `mojo`?